src/components/Cards.js

```diff
@@ -0,0 +1,47 @@
+import "./Cards.css";
+import { getDate, getElapsedTime } from "../utils";
+import { useState } from "react";
+
+function Card({ data }) {
+  const [hover, setHover] = useState(false);
```

Comment:
boolean 값들은 변수명을 지을 때 주로 is, has, should 와 같은 접두사를 붙여주는게 좋아요
```suggestion
  const [isHovered, setIsHovered] = useState(false);
```